utils/rpms/daos.spec

```
@@ -81,7 +81,7 @@ BuildRequires: libyaml-devel
 BuildRequires: libcmocka-devel
 BuildRequires: valgrind-devel
 BuildRequires: systemd
-BuildRequires: go >= 1.17
+BuildRequires: go >= 1.21
```

A bit late now I suppose since the master PR for this has already landed, but you could add a new `%global go_version 1.21`  after https://github.com/daos-stack/daos/blob/b227b4acd481bf7dd7073e80031a51f3d16aea63/utils/rpms/daos.spec#L7 and then this would be:
```suggestion
BuildRequires: go >= %{go_version}
```
and the same for https://github.com/daos-stack/daos/blob/b227b4acd481bf7dd7073e80031a51f3d16aea63/utils/rpms/daos.spec#L221 below so that you only have one place to have to update this.

Maybe next time.

---

Thanks for the tip, Brian. Yeah, we should probably make that fix at the master level first.